packages/terra-image/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Fix flickering on new props #1748.
```

Comment:
There may not be any official documentation on this, but there are a few main groups of headings for the changelog.

Changed, Added, Breaking Change, Fixed

Would you mind updating this to ### Fixed ?

Reply:
Absolutely! Fixed.